_data/chains/eip155-33626250.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Toliman Suave Testnet",
+  "chain": "ETH",
+  "rpc": ["https://rpc.toliman.suave.flashbots.net"],
+  "faucets": ["https://faucet.toliman.suave.flashbots.net"],
+  "nativeCurrency": {
+    "name": "SUAVE Toliman Eth",
+    "symbol": "TEEth",
+    "decimals": 18
+  },
+  "features": [{ "name": "EIP155" }, { "name": "EIP1559" }],
+  "infoURL": "https://suave-alpha.flashbots.net/toliman",
+  "shortName": "",
```

I guess this should make linter happy => CI pass:

```suggestion
  "shortName": "suave-toliman",
```